src/components/Navegation/Navegation.scss

```
@@ -0,0 +1,45 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+
+@font-face{
+    font-family: 'Roboto';
+    src: url(../../fonts/Roboto-Regular.ttf);
+}
+
+.Navegation{
```

No le sacas potencial a SCSS y es Navigation, no Navegation.

Refactoriza de esta manera: 
```suggestion
.Navegation{
  /* styles */
  &__li {
  /* styles */
  }
  &__image {
  /* styles */
  }
  &__ul {
    &-left-side {
    /* styles */
    }
    &-right-side {
    /* styles */
    }
  }
}
etc...
```

También, yo no le pondría "ul" o "li" como clases, sino más bien algo tipo Navigation-container, Navigation-element, etc... (esto habría que hacer una convención o algo parecido).
También en .Navigation__ul-left-side le pondría Navigation-container-leftSide (camelCase a los nombres con dos palabras que van juntas, también habría que hacer una convención o lograr encontrar algo con lo que todos estemos cómodos)